phpcs.xml

```diff
@@ -16,7 +16,10 @@
 	<config name="testVersion" value="7.0-" />
 
 	<!-- Rules -->
-	<rule ref="WooCommerce-Core" />
+	<rule ref="WooCommerce-Core">
+		<exclude name="WordPress.Files.FileName.InvalidClassFileName"/>
+		<exclude name="WordPress.Files.FileName.NotHyphenatedLowercase"/>
```

Comment:
Makes sense to me, especially if this is consistent with other cutting-edge Woo plugins (admin, blocks). 

Would be extra nice to add a comment here documenting the decision & rationale.

Reply:
Good call. I've added a comment.